server/application-server/src/main/java/de/tum/in/www1/hephaestus/codereview/repository/Repository.java

```diff
@@ -0,0 +1,60 @@
+package de.tum.in.www1.hephaestus.codereview.repository;
+
+import java.time.Instant;
+import java.util.List;
+
+import de.tum.in.www1.hephaestus.codereview.pullrequest.Pullrequest;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name = "repositories")
+@Getter
+@Setter
+@NoArgsConstructor
+public class Repository {
+
+    /**
+     * Unique identifier for a Repository
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "github_id")
+    private Long githubId;
+
+    @Column
+    private String name;
+
+    @Column(name = "name_with_owner")
+    private String nameWithOwner;
+
+    @Column
+    private String description;
+
+    @Column
+    private String url;
```

Comment:
What is this `url`? 
Is it the `clone_url` or the `html_url` or the REST API url? 

I think we should have two properties called:
`html_url` and `ssh_url`, the others are not important

Reply:
Currently it refers to the Website/HTML Url. I guess an interesting question here is if we really need any of these URLs. All of them are pretty easy to generate:
- HTML URL: `https://github.com/{nameWithOwner}`
- Clone HTTPS URL: `https://github.com/{nameWithOwner}.git`
- Clone SSH URL: `git@github.com:{nameWithOwner}.git`

Do you think we should store any of them?

Reply:
Alright, no I think it is fine the way you are doing it. We don't really need the others.